optimizely/event/events.go

```diff
@@ -62,8 +62,8 @@ type ConversionEvent struct {
 
 // LogEvent represents a log event
 type LogEvent struct {
-	endPoint string
-	event    Batch
+	EndPoint string
```

Comment:
Nice, this should definitely be public if we expect the event dispatcher to be replaceable